overlays/parallella/etc/apt/sources.list.d/parallella-ppa.list

```diff
@@ -1,5 +0,0 @@
-# Parallella PPA
```

Comment:
No biggie but can we please leave this in but replace vivid --> bionic?

There actually is a parallella ppa but we never used it.